e2e_samples/dataset_versioning/README.md

```diff
@@ -1,79 +1,67 @@
 # DataOps - Dataset versioning demo
 
-This sample demonstrates how to use a data pipeline to copy versioned data into Data Lake.
+This sample demonstrates how to [incrementally ingest data](https://docs.microsoft.com/en-au/azure/data-factory/tutorial-incremental-copy-overview) from data source such an AzureSQL Database with [Azure Data Factory Dataflows](https://docs.microsoft.com/en-us/azure/data-factory/control-flow-execute-data-flow-activity). The ingested data in Data Lake is also versioned through the use of the [Delta format](https://delta.io/).
 
 ## Story
 
 Contoso provides ML based loan prediction feature to end users. For keeping end-user experience, they want to keep updating ML model with latest data. It helps Contoso to keep end user satisfaction high and subscription-based contract.
 
-For achieving it, data scientists (DS) asks data engineers to save multiple-version data (ex. v0 data is saved in 2020, v1 data is saved in 2021). It helps DS to manage data and trained model. For saving cost, data engineers want to copy data with shorter duration.
+However, while Contoso Data Engineers requires the data in the Data Lake to be kept reasonably up to date with the source operational system, the Data Scientists on the other hand requires a fixed version of the data to train and tune a model against in order to properly benchmark the accuracy of each model. Therefore, there is a need for the ability to consume a specific version the data ingested into the data lake.
 
 ## Solution overview
 
 The solution demonstrates how to achieve the requirements described above by:
 
-- Data Pipeline (Azure Data Factory) copy **versioned data** from source to sink
-- DS can read versioned data from Data Bricks. They can specify version number when they load data from Delta Lake.
-- Data pipeline utilizes a watermark to see new/updated data in the data source. It helps to **copy data with shorter duration**.
-- Terraform helps enginners to provision expected infrastructure.
-- Local python script inject data to data source (SQL Database) incrementally, so that you can simulate data source update(ex. you can inject 2020 data at first, then inject 2021 data to data source).
+- Incrementally ingesting data from a source system such as AzureSQL Database.
+  - Data pipeline (Azure Data Factory) utilizes a watermark to see new/updated data in the data source. It helps to copy data with shorter intervals.
```

Comment:
Is this meant to indent?

Reply:
Yup, its meant to be a subpoint to the "Incrementally ingesting data".